source/sass/views/home.scss

```diff
@@ -40,3 +40,9 @@
     width: 50%;
   }
 }
+
+#view-home {
+  .h1-white {
+    font-size: 48px;
```

Comment:
This feels like not the right solution, especially since this text isn't white. Is it the right class to begin with?

Reply:
This is the correct class for the end result...we're going to change the class names post-Wagtail migration so that they are color agnostic.